### PR TITLE
In listing views, I want to see locality information if applicable

### LINKED ISSUE #1646: In listing views, I want to see locality information if applicable
eg. https://lawlibrary.org.za/taxonomy/subject-areas/subject-areas-environment-climate-and-wildlife which blends national, provincial and municipal items, it should include the locality (if any)

### PR DESCRIPTION
**Pull request by Sandravaphilips**
Closes #1646
Loom: https://www.loom.com/share/744292bf14c24d049528c369f1343583?sid=3217dbf5-1583-42f0-842c-8ec87785a200

### CODE REVIEW

File: peachjam/templates/peachjam/_document_table.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
     <thead>
       <tr>
         <th scope="col" style="width: 70%">{% trans 'Title' %}</th>
-        {% if doc_table_show_jurisdiction %}
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/_document_table.html:7]: What was the logic to determine the value of this variable previously?

File: peachjam/templates/peachjam/_document_table.html [hunk 2]
```diff
@@ -19,8 +19,12 @@
           <td>
             <a href="{{ document.get_absolute_url }}">{{ document.title }}</a>
           </td>
-          {% if doc_table_show_jurisdiction %}
-            <td>{% jurisdiction_icon document %}&nbsp;{{ document.jurisdiction }}</td>
+          {% if MULTIPLE_JURISDICTIONS and MULTIPLE_LOCALITIES %}
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/_document_table.html:22]: Not quite:

1. if `MULTIPLE_JURISDICTIONS`
  a. show document.country
  b. if document.locality, show it
2. elif `MULTIPLE_LOCALITIES`
  a. if document.locality, show it

File: peachjam/templates/peachjam/_document_table.html (hunk 1 above)
Comment (Sandravaphilips) [reply peachjam/templates/peachjam/_document_table.html:7]: It's set only in **africanlii** to show the jurisdiction of taxonomies alone. But if we're showing jurisdiction across the site, I didn't think it'd matter.

File: peachjam/templates/peachjam/_document_table.html (hunk 2 above)
Comment (Sandravaphilips) [reply peachjam/templates/peachjam/_document_table.html:22]: In the case of MULTIPLE_JURISDICTIONS, do you want the locality showing after the jurisdiction or before? Or should it be a separate column?

File: peachjam/templates/peachjam/_document_table.html (hunk 1 above)
Comment (longhotsummer) [reply peachjam/templates/peachjam/_document_table.html:7]: Agreed. Can you remove the logic that sets it in views.py or wherever?

File: peachjam/templates/peachjam/_document_table.html (hunk 2 above)
Comment (longhotsummer) [reply peachjam/templates/peachjam/_document_table.html:22]: Same column, but after. Separate with a https://en.wikipedia.org/wiki/Interpunct `·`

eg. `South Africa · Gauteng`

File: peachjam/templates/peachjam/_document_table.html (hunk 1 above)
Comment (Sandravaphilips) [reply peachjam/templates/peachjam/_document_table.html:7]: Screenshot 2024-01-22 at 11 53 32

File: peachjam/templates/peachjam/_document_table.html (hunk 2 above)
Comment (Sandravaphilips) [reply peachjam/templates/peachjam/_document_table.html:22]: Is there a difference between _if document.locality, show it_ and _document.locality|default_if_none:''_? Either way, a document without locality won't display anything

Comment (longhotsummer) [reply peachjam/templates/peachjam/_document_table.html:22]: Correct, no difference, except that you don't want the middot if there is no locality.

**Comment by Sandravaphilips** [comment]
@longhotsummer back to you

**Review by longhotsummer — APPROVED** [review]
great

**Pull request merged**